uikit/src/main/java/com/midtrans/sdk/uikit/activities/CreditCardFlowActivity.java

```diff
@@ -867,4 +881,38 @@ private void showBankPointsFragment(float balance, String bankType) {
         getSupportFragmentManager().beginTransaction().replace(R.id.card_container, fragment).addToBackStack("").commit();
     }
 
+    public boolean isClickPayment() {
```

Comment:
This will be better if we only relied on `credit_card` options on Snap so we can support `token` only on SDK